plugins/generator-1.20.4/neoforge-1.20.4/mappings/screens.yaml

```diff
@@ -2,96 +2,103 @@ _default: Ingame
 Ingame: Ingame
 MainMenu: TitleScreen
 IngameMenu: PauseScreen
+Anvil: AnvilScreen
+Beacon: BeaconScreen
+BlastFurnace: BlastFurnaceScreen
+BrewingStand: BrewingStandScreen
+CartographyTable: CartographyTableScreen
+Chest: ContainerScreen
+CommandBlock: CommandBlockEditScreen
+Crafter: CrafterScreen
+Crafting: CraftingScreen
+Dispenser: DispenserScreen
+EditBook: BookEditScreen
+EditMinecartCommandBlock: MinecartCommandBlockEditScreen
+EditSign: SignEditScreen
+EditStructure: StructureBlockEditScreen
+Enchantment: EnchantmentScreen
+Furnace: FurnaceScreen
+Grindstone: GrindstoneScreen
+HangingSignEdit: HangingSignEditScreen
+Hopper: HopperScreen
+HorseInventory: HorseInventoryScreen
+Jigsaw: JigsawBlockEditScreen
+Lectern: LecternScreen
+Loom: LoomScreen
+Merchant: MerchantScreen
+ReadBook: BookViewScreen
+ShulkerBox: ShulkerBoxScreen
+SmithingTable: SmithingScreen
+Smoker: SmokerScreen
+Stonecutter: StonecutterScreen
+AccessibilityOnboarding: AccessibilityOnboardingScreen
+AccessibilityOptions: AccessibilityOptionsScreen
 AddServer: EditServerScreen
+Advancements: AdvancementsScreen
 Alert: AlertScreen
-ChatOptions: ChatOptionsScreen
 Chat: ChatScreen
-CommandBlock: CommandBlockEditScreen
+ChatOptions: ChatOptionsScreen
+ChatReport: ChatReportScreen
+ChatSelection: ChatSelectionScreen
+Confirm: ConfirmScreen
 ConfirmBackup: BackupConfirmScreen
+ConfirmExperimentalFeatures: ConfirmExperimentalFeaturesScreen
 ConfirmOpenLink: ConfirmLinkScreen
-Confirm: ConfirmScreen
 Connecting: ConnectScreen
 Controls: ControlsScreen
 CreateBuffetWorld: CreateBuffetWorldScreen
 CreateFlatWorld: CreateFlatWorldScreen
 CreateWorld: CreateWorldScreen
+Creative: CreativeModeInventoryScreen
+CreditsAndAttribution: CreditsAndAttributionScreen
 CustomizeSkin: SkinCustomizationScreen
 DatapackFailure: DatapackLoadFailureScreen
 Death: DeathScreen
 Demo: DemoIntroScreen
 DirtMessage: GenericDirtMessageScreen
 Disconnected: DisconnectedScreen
 DownloadTerrain: ReceivingLevelScreen
-EditBook: BookEditScreen
 EditGamerules: EditGameRulesScreen
-EditMinecartCommandBlock: MinecartCommandBlockEditScreen
-EditSign: SignEditScreen
-EditStructure: StructureBlockEditScreen
 EditWorld: EditWorldScreen
-Enchantment: EnchantmentScreen
+EffectRenderingInventory: EffectRenderingInventoryScreen
 Error: ErrorScreen
+Experiments: ExperimentsScreen
 FlatPresets: PresetFlatWorldScreen
 GamemodeSelection: GameModeSwitcherScreen
+GenericWaiting: GenericWaitingScreen
 GPUWarning: PopupScreen

```

Comment:
This no longer appears to be present, so we should remove it from the new mappings.
```suggestion

```

Reply:
And also from current generator mapping files + datalist

Reply:
_This_ being the only one removed or I should check for more myself?

Reply:
This is the only one. Matej ran it through porting tools

Reply:
I think this is fine as is.